shop_richcatalog/cms_plugins.py

```diff
@@ -11,7 +12,16 @@ class CMSCatalogPlugin(CMSPluginBase):
     render_template = "catalog/catalog_grid_plugin.html"
 
     def render(self, context, instance, placeholder):
-        context['catalog_list'] = Catalog.objects.all()
+        context = super(CMSCatalogPlugin, self).render(
+            context, instance, placeholder)
```

Comment:
We don't technically need the super call, but the django cms plugin documentation suggests to populate the context this way if you are overwriting the render method.